src/Components/Artwork/Metadata.tsx

```diff
@@ -2,6 +2,7 @@ import { Metadata_artwork } from "__generated__/Metadata_artwork.graphql"
 import { ContextConsumer } from "Artsy/Router"
 import colors from "Assets/Colors"
 import { garamond } from "Assets/Fonts"
+import StyledTextLink from "Components/TextLink"
```

Comment:
What is this component?

Reply:
This makes the tombstone info be a plain link, but without all this other 'cruft', like underlining on hover.

Making it `<a>` seemed to add a bunch of that stuff, and I couldn't figure out how to remove that. So this actually works great!